tests/integration_tests/functional/test_drives.py

```diff
@@ -337,6 +338,134 @@ def test_patch_drive(test_microvm_with_ssh, network_config):
     assert stdout.readline().strip() == size_bytes_str
 
 
+def test_no_flush(test_microvm_with_ssh, network_config):
+    """Verify default block ignores flush."""
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    test_microvm.basic_config(
+        vcpu_count=1,
+        add_root_device=False
+    )
+
+    _tap, _, _ = test_microvm.ssh_network_config(network_config, '1')
+
+    # Add the block device
+    test_microvm.add_drive(
+        'rootfs',
+        test_microvm.rootfs_file,
+        root_device=True,
+    )
+
+    # Configure the metrics.
+    metrics_fifo_path = os.path.join(test_microvm.path, 'metrics_fifo')
+    metrics_fifo = log_tools.Fifo(metrics_fifo_path)
+    response = test_microvm.metrics.put(
+        metrics_path=test_microvm.create_jailed_resource(metrics_fifo.path)
+    )
+    assert test_microvm.api_session.is_status_no_content(response.status_code)
+
+    test_microvm.start()
+
+    # Verify no flush commands were generated during boot.
```

Comment:
actually, the commands are generated but they are ignored

Reply:
Indeed, changed it.